charts/nri-metadata-injection/Chart.yaml

```diff
@@ -9,8 +9,8 @@ sources:
   - https://github.com/newrelic/k8s-metadata-injection
   - https://github.com/newrelic/k8s-metadata-injection/tree/master/charts/nri-metadata-injection
 
-version: 4.0.0
-appVersion: 1.7.5
+version: 4.0.1
```

Comment:
Let's do a minor bump for both of these versions. I saw a comment here saying dependencies should be a minor bump: https://github.com/newrelic/nri-kubernetes/pull/617#discussion_r1101481802

Reply:
updated to minor bump